Vehicle.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CopilotCodeReviewDemo
+{
+    internal class Vehicle : IVehicle
```

Comment:
Property names should follow PascalCase naming convention. Consider renaming `model` to `Model` and `year` to `Year` for consistency.